pycheribuild/projects/build_qemu.py

```diff
@@ -314,7 +323,42 @@ def setup(self):
             ])
 
 
-class BuildMorelloQEMU(BuildQEMU):
+class BuildBsdUserQEMU(BuildQEMUBase):
+    repository = GitRepository("https://github.com/CTSRD-CHERI/qemu.git",
+                               default_branch="qemu-cheri-bsd-user",
+                               force_branch=True)
+    native_install_dir = DefaultInstallDir.BSD_USER_SDK
```

Comment:
You shouldn't need a whole new enum for this, just copy upstream-qemu's approach?

Reply:
Does this install anything other than the qemu-user binaries? If not we could reuse the same installation directory?